starter-code/src/functions-and-arrays.js

```diff
@@ -99,3 +185,43 @@ const matrix = [
   [20, 73, 35, 29, 78, 31, 90, 1, 74, 31, 49, 71, 48, 86, 81, 16, 23, 57, 5, 54],
   [1, 70, 54, 71, 83, 51, 54, 69, 16, 92, 33, 48, 61, 43, 52, 1, 89, 19, 67, 48]
 ];
+
+function greatestProduct(productArray) {
+  let mergedArray = [].concat(...productArray);
```

Comment:
Woah, smart stuff! By concatenating the arrays and making it one dimensional, you won't need to run a nested loop. Very very cool :D

Reply:
This article was very helpful in explaining the steps: https://blog.hellojs.org/projecteuler-net-8-11-in-javascript-79508138731b